app/qml/components/MMListMultiselectDrawer.qml

```diff
@@ -52,7 +52,8 @@ MMDrawer {
 
         placeholderText: qsTr( "Search" )
 
-        bgColor: __style.lightGreenColor
+        textFieldBackground: __style.lightGreenColor
```

Comment:
Shouldn't it be like this? 🤔  `textFieldBackground` aliases to Rectangle, not its color
```suggestion
        textFieldBackground.color: __style.lightGreenColor
```